js/src/Components/NLPInferencePage.tsx

```diff
@@ -13,16 +13,21 @@ interface InferecePageProp {
 
 const defaultPhrase = "Serverless ";
 
+var Filter = require('bad-words'),
```

Comment:
same comment in js/src/Components/NLPGenerator.tsx

Reply:
Taken care of by above.